src/ripple/rpc/impl/RPCHandler.cpp

```diff
@@ -65,7 +65,7 @@ namespace {
      Failure:
         {
            "result" : {
-              "error" : "noNetwork",
+              "error" : "notSynced",
               "error_code" : 16,
               "error_message" : "Not synced to Ripple network.",
```

Comment:
Let's change this message to reflect the message change in ErrorCodes.cpp

Reply:
Done.